awx/api/serializers.py

```diff
@@ -4873,7 +4873,7 @@ class InstanceGroupSerializer(BaseSerializer):
 
     show_capabilities = ['edit', 'delete']
 
-    committed_capacity = serializers.SerializerMethodField()
+    committed_capacity = serializers.SerializerMethodField(help_text=_('This resource has been deprecated and will be removed in a future release'))
     consumed_capacity = serializers.SerializerMethodField()
```

Comment:
fwiw consumed_capacity also uses `capacity_values` in the `ha.py` that we've talked about axing. We should think about if we can deprecate that too (unless we want to do some other re-implementation w/o the capacity_values function)

Reply:
I don't think we should want to remove the IG `consumed_capacity`. Ultimately, if we cache `task_impact` for unified jobs, then we can do the summation in the database by using an efficient Django annotate.